scalastyle-config.xml

```diff
@@ -86,7 +86,7 @@ This file is divided into 3 sections:
   </check>
 
   <check level="error" class="org.scalastyle.scalariform.ObjectNamesChecker" enabled="true">
-    <parameters><parameter name="regex"><![CDATA[[A-Z][A-Za-z]*]]></parameter></parameters>
+    <parameters><parameter name="regex"><![CDATA[(config|[A-Z][A-Za-z]*)]]></parameter></parameters>
```

Comment:
Agree it makes more sense to make the whole thing `private[spark]`, but couldn't you just turn the rule off around this object, like we do for println?

(Also I'm confused why `package object config` doesn't violate the rule).